R/assert_is_unique_dt.R

```diff
@@ -0,0 +1,47 @@
+#' Are the rows of the input data.table unique?
+#'
+#' Checks to see if the rows of the input data.table are unique for each
+#' combination of the id columns.
```

Comment:
Need to add 2 spaces after the first line of the description, unless the `@description` tag.

Ex:
```
blah blah blah
  second line of blah
```
or
```
@description 
blah blah
blah blah blah
```

Reply:
@spencerpease Are you saying this is needed to prevent the second line from going into `@details` or just stylistically?

Reply:
See other follow up comment